sage_modabvar/abvar.py

```diff
@@ -646,15 +646,18 @@ def elliptic_curve(self):
             raise RuntimeError("Elliptic curve not found" +
                                " in installed database")
 
-        sturm = ModularSymbols(N, 2).sturm_bound()
         a = lambda p: f.modular_symbols(1).eigenvalue(p, 'a')
```

Comment:
On Tuesday, July 19, 2016, Kevin Lui <notifications@github.com> wrote:

> In sage_modabvar/abvar.py
> <https://github.com/williamstein/sage_modabvar/pull/29#discussion_r71456144>
> :
>
> > @@ -646,15 +646,18 @@ def elliptic_curve(self):
> >              raise RuntimeError("Elliptic curve not found" +
> >                                 " in installed database")
> >
> > -        sturm = ModularSymbols(N, 2).sturm_bound()
> >          a = lambda p: f.modular_symbols(1).eigenvalue(p, 'a')
>
> Just a remark: Computing ModularSymbols(N, 2).sturm_bound() isn't hard --
> it's a trivial arithmetic function of the weight and level. What is hard is
> "computing [Hecke eigenvalues] up to the Sturm bound for f...".
>
> Okay. The a(p) code gets called once for each elliptic curve so I checked
> that the result of a(p) was cached.
>

Also computing ap for an elliptic curve is super fast.  100000x faster than
for modabvars...



> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/williamstein/sage_modabvar/pull/29/files/00eb180d198a6703ab02fd05818071ca7971a759#r71456144>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABN5dn_SMGbU7UKz9rEhjkc8MZ17JANNks5qXYUEgaJpZM4JQSfR>
> .
>


-- 
Sent from my massive iPhone 6 plus.